images/linux/scripts/installers/containers.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+################################################################################
+##  File:  containers.sh
+##  Desc:  Installs container tools: podman, buildah and skopeo onto the image
+################################################################################
+
+source $HELPER_SCRIPTS/apt.sh
+source $HELPER_SCRIPTS/document.sh
+
+source /etc/os-release
+sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/x${NAME}_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
```

Comment:
I have tried to install on Ubuntu 16.04:
https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_16.04/Release.key:
2020-04-07 10:18:24 ERROR 404: Not Found.
##[error]Process completed with exit code 8.

Reply:
The tools are likely not supported on Ubuntu 16.04.

Reply:
@kpfleming, You are right this repo doesn't contain packages for  `xUbuntu_16.04` only for `xUbuntu_18.04` and higher - https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/

Looks like we should find another repo like:
https://launchpad.net/~projectatomic/+archive/ubuntu/ppa

Reply:
> @kpfleming, You are right this repo doesn't contain packages for `xUbuntu_16.04` only for `xUbuntu_18.04` and higher - https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/
> 
> Looks like we should find another repo like:
> https://launchpad.net/~projectatomic/+archive/ubuntu/ppa

It looks like if PPA is abandoned, there is no activity for 20 weeks. And the official instructions points to this kubic repo https://podman.io/getting-started/installation .
And yes... 16.04 is not supported... Probably I should move these changes to 18.04 only.

Reply:
@kpfleming @al-cheb I removed changes for 16.04.